example/dj/urls.py

```diff
@@ -1,5 +1,7 @@
+from distutils.version import StrictVersion
+
 import django
-from django.conf.urls import patterns, url
+from django.conf.urls import url
```

Comment:
To je dobře, dělat na to nějakou datovou strukturu místo seznamu bylo hodně Javovský:D